src/app/static/src/tests/components/workflowWizard/workflowWizard.test.ts

```diff
@@ -16,7 +17,7 @@ describe(`workflowWizard`, () => {
     const getWrapper = (mockStep = steps) => {
         return mount(workflowWizard, {
                 propsData: {
-                    initialRunWorkflowMetadata: {placeholder: "testdata"},
+                    initialRunWorkflowMetadata: {...emptyWorkflowMetadata},
```

Comment:
This test suite is effectively integration testing with the real child components, so here we do feed them the workflow metadata they require to initialise.